tools/GSDumpGUI/GSDumpGUI.csproj

```diff
@@ -10,14 +10,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>GSDumpGUI</RootNamespace>
     <AssemblyName>GSDumpGUI</AssemblyName>
-    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Is there a need to up the framework version, if it's not required then we can skip it.

Reply:
I originally wanted to use a nuget package but was lost in licence hell and wrote the code myself. I can put it back.

Reply:
Got the same result. 4.0 is minimum now.

I think most people have at least 4.7 on their machines so I don't think anyone is negatively affected here.

Reply:
> 4.0 is compatible to XP in case anyone cares.

I don't think anyone cares at this point :)

Btw I'm on 4.6.1 so not really on latest.